AirCasting/SearchAndFollow/CompleteScreen/Views/CompleteScreenViewModel.swift

```diff
@@ -141,6 +142,43 @@ class CompleteScreenViewModel: ObservableObject {
         exitRoute()
     }
     
+    private func getAndDisplayData() {
+        if session.stream.first!.sensorName.contains("AirBeam") {
+            do {
+                try streamsDownloader.downloadStreams(with: session.id, for: .airBeam3) { result in
+                    switch result {
+                    case .success(let success):
+                        // TODO: - FIX Thresholds and Improve this part of code
```

Comment:
Is this PR ready for review with this Thresholds logic that still needs to be added? I think we may need to think of another approach for this task, like calling a different end point, if we are not getting threshold values from this one.